blt-project/blt/sites/default.yml

```diff
@@ -0,0 +1,12 @@
+# @file default.yml
+# Site definition file for the default site.
+aliases:
```

Comment:
My latest commits have made all configuration under `blt/sites/*` optional, with the values set in `project.yml` being used if no site-specific configuration is supplied.